gigamon/logs/Gigamon.yml

```diff
@@ -0,0 +1,128 @@
+id: gigamon
+# See app_id in your integration's manifest.json file to learn more:
+# https://docs.datadoghq.com/developers/integrations/check_references/#manifest-file
+metric_id: gigamon
+# If for some reason id must be different than app_id, add the app_id in this field instead.
+# If id and app_id already match, this field can be left blank.
```

Comment:
You can follow these comments and remove the `metric_id` since it's the same as the `id` and clean up the comments too